_compendium_entries/2020-10-02.md

```diff
@@ -21,5 +21,6 @@ In preparation for our next meeting, (if you’re interested in coming!), please
 1. [How to create a GitHub account (with pictures)](https://www.wikihow.com/Create-an-Account-on-GitHub){: target="_blank" .entry-link}
 2. [How to download and setup Visual Studio Code](https://code.visualstudio.com/docs/introvideos/basics){: target="_blank" .entry-link}
 3. [Good practice for file trees when developing](https://www.sitepoint.com/organize-project-files/){: target="_blank" .entry-link}
-4. Kanye fun fact #3:  kanye doesn't like water bottles. in fact he hates them, especially on airplanes when they just show up. ![kanye west tweet](assets/images/contentImages/kanyefact3.png)
+4. Kanye fun fact #3:  kanye doesn't like water bottles. in fact he hates them, especially on airplanes when they just show up. 
+<br>![kanye west tweet](assets/images/contentImages/kanyefact3.png)
```

Comment:
Also, can we keep this? Right now the picture is pushing down the last line of text, but the <br> fixes that.
![image](https://user-images.githubusercontent.com/56608573/95326852-38ab2900-0871-11eb-9a33-54711964c470.png)

Reply:
Yes that's good

Reply:
That'd be great if you could reopen this one!

Good to know! I'll use the backslash instead instead.